Source/Core/DolphinQt/Settings/AdvancedPane.cpp

```diff
@@ -264,6 +311,30 @@ void AdvancedPane::Update()
     return tr("%1% (%2 MHz)").arg(QString::number(percent), QString::number(clock));
   }());
 
+  QFont vi_bf = font();
+  vi_bf.setBold(Config::GetActiveLayerForConfig(Config::MAIN_VI_FREQUENCY_ENABLE) !=
+                Config::LayerType::Base);
+  m_vi_rate_override_checkbox->setFont(vi_bf);
+  m_vi_rate_override_checkbox->setChecked(enable_vi_rate_override_widgets);
+
+  m_vi_rate_override_slider->setEnabled(enable_vi_rate_override_widgets);
+  m_vi_rate_override_slider_label->setEnabled(enable_vi_rate_override_widgets);
+
+  {
+    const QSignalBlocker blocker(m_vi_rate_override_slider);
+    m_vi_rate_override_slider->setValue(
+        static_cast<int>(std::round(Config::Get(Config::MAIN_VI_FREQUENCY) * 100.f)));
+  }
+
+  m_vi_rate_override_slider_label->setText([] {
+    int percent = static_cast<int>(std::round(Config::Get(Config::MAIN_VI_FREQUENCY) * 100.f));
+    float vps =
+        static_cast<float>(Core::System::GetInstance().GetVideoInterface().GetTargetRefreshRate());
+    if (vps == 0.0f || !Config::Get(Config::MAIN_VI_FREQUENCY_ENABLE))
```

Comment:
Do we need to check against an epsilon here instead of exactly testing 0?  Is a vps of 0 even possible?

Reply:
It's in case VI hasn't been initialized yet, the default value is 0.

https://github.com/dolphin-emu/dolphin/blob/d2db9d95906de73d704bd0f32bd1ee79f4a442f6/Source/Core/Core/HW/VideoInterface.h#L436